PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1479,11 +1479,11 @@ void Notepad_plus::removeEmptyLine(bool isBlankContained)
 	// remove the last line if it's an empty line.

```

Comment:
Why not just check the last line's content to decide to remove it, rather than running a regex replace over the entire doc?

Reply:
>Can you provide some code example

I don't currently have any such example. :-(